uni/lib/json.icn

```diff
@@ -991,3 +998,233 @@ end
 ########################
 # END PARSER FUNCTIONS #
 ########################
+
+link fullimag
+
+
+#<p>
+# validate a json file using a json sechema
+# <[param jfname - the name of the json File]>
+# <[param sfname - the name of the json Schema File]>
+#</p>
+#
+procedure jsf_validate(jfname, sfname, strict, error)
+   local json_date, json_schema
+
+   jerror := ErrorHandler(error,strict)
+
+   json_data   := jtou(jfname , "f", strict, error)
+   json_schema := jtou(sfname , "f", strict, error)
+
+   return _jsvalidation(json_data, json_schema, json_schema)
+end
+
+procedure jss_validate(data, schema, strict, error)
```

Comment:
Need to document the parameters. (I'm guessing that this is "string validate" as opposed to "file validate".  Also make the same fix to `jerror` as above (if needed).